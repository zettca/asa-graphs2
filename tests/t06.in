30 50
9
28 18 1
7 27 -1
8 24 -1
26 23 -2
9 14 -2
7 4 3
13 1 -2
5 6 1
14 8 1
30 12 1
30 5 1
2 21 0
4 23 3
6 9 0
15 17 -3
12 10 0
24 18 1
30 2 2
21 1 2
6 20 0
22 1 -1
15 9 1
26 25 3
19 16 2
20 19 -1
29 10 -1
3 11 0
6 19 -1
18 8 3
29 7 2
14 4 -2
16 28 1
17 22 0
10 11 -2
29 20 0
1 9 2
22 4 3
16 8 -2
25 29 -3
6 18 3
5 27 -3
2 24 2
3 28 -3
15 23 2
28 17 3
8 12 1
11 14 -1
2 17 -2
11 7 -3
6 23 0
