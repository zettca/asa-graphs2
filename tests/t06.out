U
U
U
I
U
U
I
I
0
I
I
I
U
I
U
U
U
I
U
U
U
U
I
I
U
U
I
U
U
U
